mmedit/models/losses/perceptual_loss.py

```diff
@@ -202,3 +206,59 @@ def _gram_mat(self, x):
         features_t = features.transpose(1, 2)
         gram = features.bmm(features_t) / (c * h * w)
         return gram
+
+
+@LOSSES.register_module()
+class TransferalPerceptualLoss(nn.Module):
```

Comment:
Double check Texture vs Transferal. If the latter, pls modify the pr title accordingly